src/debug/fake-flow.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useCallback } from 'react';
```

Comment:
This component won't be in the final app, it's just to give a helpful window into the state interaction for now